entries/entry04.md

```diff
@@ -1,7 +1,31 @@
 # Entry 4
-##### X/X/XX
+##### 3/8/20
+### **Sources:**
+
+I am still using the Tinkercad Arduino Board simulator. Using this simulator, I can use an artificial Arduino Board for free. This simulator is an efficient way to test my project because I dont have to spend minutes storing away physical components. The Software provies all of Arduino's gadgets and tools.  
+
+**Links:**  
```

Comment:
As I've mentioned, don't just plop links in your entry without any context. **Show** exactly _how_ you've used these sources.